CRM/Core/EntityTokens.php

```diff
@@ -138,9 +138,23 @@ public function evaluateToken(TokenRow $row, $entity, $field, $prefetch = NULL)
         Civi::log()->info('invalid date token');
       }
     }
+    if ($this->isHTMLTextField($field)) {
```

Comment:
So we are interpretting the field metadata to tell the token system if we are giving it html or plain text